### DISCUSSION TITLE
Welcome to audacity_tools_for_blender Discussions!

### DISCUSSION THREAD

**Discussion by tin2tin** [tin2tin/Audacity_Bridge_for_Blender#10] [Ideas]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, start a new thread in the right category.

**Comment by in10siv** [comment]
Hi, seems that i'm the first so here it is:
 1 when u receive a mixdown there is no mute to the individual strips.
 2 there is a specific way that you can get a python error if the selected strip was deleted and you try to send the strip. it's quite an easy fix.
 
next i have a suggestion for a new button/function to get the receive the mixdown linked to a speaker object so you can add the dimensionality of your sound effects.
or even better the implementation of these ideas : https://developer.blender.org/T84944#1103168

Great addon !

**Comment by tin2tin** [comment]
1. I think you should to this manually. Maybe a t some point I want to add a Selected mode, which is like sequence, but for the selected files and in that case you can't be sure everything needs to be muted when receiving. Users could also have added additional files in blender while editing in Audacity.
2. That check might have been lost along the way. Thanks for the heads up.  

I would prefer to keep the 3D space/surround out of this. At least for now. There are plenty of thing to iron out already.

**Reply by in10siv** [reply]
thanks for the reply.  
for the 3d space i was just looking for a receive mixdown in the speaker object properties menu so you could simply get the wav file opened in the selected speaker,  it's almost the same operator.

looking forward to the evolution of this.

**Reply by tin2tin** [reply]
> for the 3d space i was just looking for a receive mixdown in the speaker object properties menu so you could simply get the wav file opened in the selected speaker, it's almost the same operator.

This is unrelated to integrating Blender and Audacity.

**Comment by tin2tin** [comment]
I made a snippet for you, to open the source file of a sound strip as a speaker object: https://gist.github.com/tin2tin/9b24dff1ab4294e9562724614e450599

(And good luck with your project, as mentioned I think this is unrelated to the Audacity project)

**Reply by in10siv** [reply]
Wow thanks a lot! It was just a suggestion i wasn't expecting for anything as long as it's not related to the project. Thanks again for the time.
